test/lib/mayaUsd/nodes/testProxyShapeBase.py

```diff
@@ -44,6 +48,134 @@ def testBoundingBox(self):
         bboxSize = cmds.getAttr('Cube_usd.boundingBoxSize')[0]
         self.assertEqual(bboxSize, (1.0, 1.0, 1.0))
 
+    def testDuplicateProxyStageAnonymous(self):
+        '''
+        Verify stage with new anonymous layer is duplicated properly.
+        '''
+        cmds.file(new=True, force=True)
+
+        # create a proxy shape and add a Capsule prim
+        import mayaUsd_createStageWithNewLayer
+        mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+        proxyShapePath = ufe.PathString.path('|stage1|stageShape1')
+        proxyShapeItem = ufe.Hierarchy.createItem(proxyShapePath)
+        proxyShapeContextOps = ufe.ContextOps.contextOps(proxyShapeItem)
+        proxyShapeContextOps.doOp(['Add New Prim', 'Capsule'])
+
+        # proxy shape is expected to have one child
+        proxyShapeHier = ufe.Hierarchy.hierarchy(proxyShapeItem)
+        self.assertEqual(1, len(proxyShapeHier.children()))
+
+        # child name is expected to be Capsule1
+        self.assertEqual(str(proxyShapeHier.children()[0].path().back()), "Capsule1")
```

Comment:
Minor: can be just proxyShapeHier.children()[0].nodeName().  In a couple of other places as well.